### ISSUE TITLE
Reminder to update data.table despite being up to date

### ISSUE THREAD

**Issue by MichaelChirico** [Rdatatable/data.table#2971]
It so happens that there's been no new commit to `dev` in over a month, so this message from `onLoad` is misleading:

```
data.table 1.11.5 IN DEVELOPMENT built 2018-06-02 00:09:06 UTC; travis  Latest news: http://r-datatable.com
**********
This development version of data.table was built more than 4 weeks ago. Please update: data.table::update.dev.pkg()
**********
```

Not sure if there's anything to do besides maybe reword (?"Please try to update"?)

**Comment by mattdowle** [comment]
Committed now so should go away.  Hopefully we won't have a 4 week pause again.
